src/web/components/ClickToView.tsx

```diff
@@ -159,39 +165,34 @@ export const ClickToView = ({
 				<div
 					className={css`
 						${textSize}
-						a {
-							${textSize}
-						}
-						p {
-							margin-bottom: 8px;
+						margin-bottom: 19px;
+						strong {
+							font-weight: bold;
 						}
 					`}
 				>
 					{source ? (
 						<>
-							<p>
-								This article includes content provided by{' '}
-								{source}. We ask for your permission before
-								anything is loaded, as they may be using cookies
-								and other technologies.
-							</p>
-							<p>
-								To view this content, click &apos;Allow and
-								continue&apos;.
-							</p>
+							This article includes content provided by {source}.
```

Comment:
Is there an accessibility impact to removing the p tags? I'm not certain but I think it might be useful to maintain the semantics of text being in a p tag.